### PR TITLE
feat(landing page): create blogs section

### LINKED ISSUE #16: Blog Section
**Create a Blog Section for the Landing Page**: 

**Description**:
- **Task**: Develop a section for featuring blog posts on the "Let's Share" landing page.
- **Objective**: Design and implement a section that displays engaging blog content to inform and engage users.

**Subtasks**:
1. **Blog Section Design**: Create a design concept for the blog section using TailwindCSS.
   - *Description*: Define the layout, content, and visual elements of the blog section with appropriate TailwindCSS classes.

2. **Blog Section Development**: Implement the blog section according to the design concept using Next.js components and TailwindCSS classes.
   - *Description*: Create Next.js components for the blog section and apply TailwindCSS classes to style the section.

3. **Responsive Design**: Ensure the blog section is responsive for various screen sizes, including mobile devices.
   - *Description*: Implement responsive design techniques using TailwindCSS classes to adapt the blog section to different screen dimensions.

4. **Snapshot Testing**: Implement snapshot testing for the Blog Section component using Jest.
   - *Description*: Write snapshot tests to capture and compare the rendering of the Blog Section component to ensure rendering consistency.

5. **Create Dummy Blog Data**: Create an array of dummy blog post objects to temporarily populate the blog section until actual blog data is available.
   - *Description*: Generate sample blog post data for initial development and testing purposes.

6. **Blog Post Component Creation**: Develop a BlogPostCard component for displaying individual blog posts with Next.js and TailwindCSS.
   - *Description*: Create a reusable Next.js component to represent blog posts, including relevant information and styling using TailwindCSS classes.

7. **Blog Post Snapshot Testing**: Implement snapshot testing for the BlogPost component using Jest.
   - *Description*: Write snapshot tests to capture and compare the rendering of the BlogPostCard component to ensure consistency.

8. **Data Mapping and Display**: Map over the data array and display blog posts in the blog section.
   - *Description*: Dynamically render blog posts in the blog card section using the Next.js BlogPost component with TailwindCSS styling based on the provided data array.

**Acceptance Criteria**:
- [x] The blog section effectively displays engaging blog posts.
- [x] The blog section is functional and responsive on various screen sizes, including mobile devices.
- [x] Snapshot tests for the Blog Section and BlogPost component are implemented and passed successfully.
- [x] An array of dummy blog post data is created and displayed in the blog section.
- [x] The BlogPostCard component is developed and correctly displays blog post information.

**Definition of Done**:
- [x] All subtasks are completed.
- [x] Code changes are reviewed and approved by the team.
- [x] The blog section on the landing page features engaging blog posts.
- [x] Snapshot tests are in place and validate the Blog Section and BlogPost component.
- [x] Dummy blog post data is successfully displayed in the blog section.
- [x] The BlogPostCard component is functional and correctly displays blog post information.

**Screen Shots**:

-Desktop:

Image

-Mobile:

Image

### PR DESCRIPTION
**Pull request by liliumorion**
Create blogs section using a reusable component (blog card) and the appropriate tests.

fixes #16
desktop view:
image

mobile view: 
image

### CODE REVIEW

**Review by riadhmouamnia — APPROVED** [review]
## Approval Review for "Blogs Section" PR

**Reviewer:** @riadhmouamnia 

Outstanding work on the "Blogs Section" PR! The section is exceptionally well-crafted, and the entire section is not only well-displayed but also highly responsive on all devices, which is a crucial aspect of a user-friendly experience.

I appreciate the inclusion of test files and reusable BlogCard components, which are essential for maintaining the reliability of our project.

Wait for another Approval review from a teammate and this PR will be ready for merging. Exceptional job! 🚀

@riadhmouamnia

**Review by rahemSorour — APPROVED** [review]
Amazing job hadia the cards looks perfect

**Pull request merged**
